offset_manager_test.go

```diff
@@ -0,0 +1,305 @@
+package sarama
+
+import (
+	"testing"
+	"time"
+)
+
+var (
+	broker, coordinator *mockBroker
+	testClient          Client
+	config              *Config
+)
```

Comment:
@eapache Do we usually have shared state like this for tests? Or do we always keep all state contained inside the test function? Not sure what to think about this.